browser/ui/BUILD.gn

```diff
@@ -1306,9 +1312,12 @@ source_set("ui") {
       "webui/brave_wallet/wallet_page_ui.h",
       "webui/brave_wallet/wallet_panel_ui.cc",
       "webui/brave_wallet/wallet_panel_ui.h",
+      "webui/email_aliases/email_aliases_bubble_ui.cc",
```

Comment:
per the comment above new files should not be added to this target. Please create a new target and use allow_circular_includes_from as necessary